corehq/apps/toggle_ui/templates/toggle/flags.html

```diff
@@ -27,6 +27,10 @@
       {% trans "Show usage metrics" %}
     </a>
   {% endif %}
+  <button id="export-button" data-bind="click: downloadFile" class="ko-template btn btn-default btn-xs pull-right">
+    <i class="fa fa-info-circle"></i>
```

Comment:
`fa-cloud-download` for consistency with other download buttons

Reply:
@orangejenny well spotted! Didn't even notice! (copy-paste)